sql/core/src/test/resources/sql-tests/results/subquery/scalar-subquery/scalar-subquery-count-bug.sql.out

```diff
@@ -106,14 +106,14 @@ from l
 -- !query schema
 struct<a:int,b:decimal(2,1),scalarsubquery(a):boolean>
 -- !query output
-1	2.0	NULL
```

Comment:
These results are incorrect. They should be `NULL` instead of `false`.

Reply:
Thanks for remind. I fixed this bug and add test to cover it. I will update `sql.out` later.